Assets/Mirror/Runtime/SyncList.cs

```diff
@@ -84,13 +85,15 @@ struct Change
         protected virtual T DeserializeItem(NetworkReader reader) => default;
 
 
-        protected SyncList()
+        protected SyncList(IEqualityComparer<T> comparer = null)
```

Comment:
why not do comparer = EquialityComparer<T>.Default by default?
would avoid the ?? stuff below

Reply:
You can only use Constant values as defaults afaik